yarn.lock

```diff
@@ -915,6 +1011,87 @@
     core-js-compat "^3.25.1"
     semver "^6.3.0"
 
+"@babel/preset-env@^7.20.2":
```

Comment:
The diff from https://github.com/DFE-Digital/early-careers-framework/pull/2719/files is quite different than here for babel, so not sure why though 🤔 ?